### PR TITLE
Update Code of Conduct using https://publiclab.org/conduct

### LINKED ISSUE #4730: Update Code of Conduct using https://publiclab.org/conduct
In recent days, we have opened FTO's in all our repositories for adding Code of Conduct and all this time we were copying the plots2/code_of_conduct file for updating other repos. So, let's update our Code_of_conduct here at plots2 first and after that copy that single copy at all other repos. 

Also, please make a list of changes which would be required to make by contributors after the document update to copy it to other repositories. 

Here's the Code of conduct at plots2 - https://github.com/publiclab/plots2/blob/master/CODE_OF_CONDUCT.md and it needs update as per https://publiclab.org/conduct .

Thanks!

### PR DESCRIPTION
**Pull request by CleverFool77**
Fixes #4730 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by gauravano** [comment]
@jywarren how about this? Also, we need to update this at all other repos after this. Abd, there are some PRs for updating the Code of Conduct already on hold, one can see those PRs tagged at #4730. Thanks!

**Comment by jywarren** [comment]
LOVE this! AWESOME! +1000000000

Thanks so much both of you!!!

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,113 +1,11 @@
-This document was copied from its home at https://publiclab.org/conduct on October 12, 2017. See that page if you would like to submit your concerns in a safe, completely anonymous way, and to learn more about this document. 
-
-****
-
 # Public Lab Code of Conduct 
 
 _Public Lab, PO Box 426113, Cambridge, MA 02142_
```
Comment (gauravano) [inline CODE_OF_CONDUCT.md]: Hey @jywarren isn't this address outdated one?

Comment (gauravano) [reply CODE_OF_CONDUCT.md]: I see Providence address - Public Lab, 55 Cromwell Street, 1C, Providence, RI 02907 at publiclab.org/conduct. So, should I update the address or remove it?

Thanks!

**Pull request merged**